skrifa/src/scale/mod.rs

```diff
@@ -246,6 +246,24 @@ mod tests {
         );
     }
 
+    #[test]
+    fn overlap_flags() {
+        let font = FontRef::new(font_test_data::VAZIRMATN_VAR).unwrap();
+        let mut cx = Context::new();
+        let mut path = scaler_test::Path {
+            elements: vec![],
+            is_cff: false,
+        };
+        let mut scaler = cx.new_scaler().build(&font);
+        let glyph_count = font.maxp().unwrap().num_glyphs();
+        for gid in 0..glyph_count {
+            let metrics = scaler.outline(GlyphId::new(gid), &mut path).unwrap();
+            // GID 2 is a composite glyph with the overlap bit on a component
+            // GID 3 is a simple glyph with the overlap bit on the first flag
+            assert_eq!(metrics.has_overlaps, gid == 2 || gid == 3);
+        }
```

Comment:
Same preference for how to assert this as #582

Reply:
Aye